dvc/command/pipeline.py

```diff
@@ -167,8 +167,11 @@ def run(self):
 
         pipelines = self.repo.pipelines
         for p in pipelines:
-            stages = networkx.get_node_attributes(p, "stage")
-            for stage in stages:
+
+            # Get the stages in order to be displayed
+            stages = list(networkx.topological_sort(p))
+
+            for stage in reversed(stages):
```

Comment:
Would it make sense to reverse the pipeline and then do the tsort? Might be easier to grasp what is going on when talking about the order of execution.

Reply:
https://github.com/iterative/dvc/pull/2928#discussion_r357391088